CoastalResilienceNetwork/tnc-network-site
# Update config + small UI changes

## Overview

Updates the site config with the real regions. Also makes some other small UI changes. See commit messages for details.

Connects #7 
Connects #9 

## Demo

![tnc1](https://user-images.githubusercontent.com/1042475/32236522-cb62f3e4-be38-11e7-8802-e04aaecf4495.gif)

## Testing Instructions

- Browse the sidebar and verify that all of the regions currently on `maps.coastalresilience.org` are in the sidebar.
- Verify that clicking a subregion launches the site.
